### PR TITLE
Check if pref survey has already been submitted.

### PR DESCRIPTION
**Pull request by AndrewPolanyi**
Small changes to check whether the preference survey is already submitted by a user.
Jayden helped change the schema to grab preferences of the logged in user.
also..from popular opinion, i remove the uvic styling from nav

image

### CODE REVIEW

File: src/pages/Survey.tsx [hunk 1]
```diff
@@ -71,6 +74,12 @@ export const Survey = () => {
     setCourseRatings([]);
   };
 
+  const checkSubmit = () => {
```
Comment (nicudy) [inline src/pages/Survey.tsx:77]: What is the purpose of these lines?

Comment (nicudy) [inline src/pages/Survey.tsx]: What is the purpose of continuously calling this function?

**Review by nicudy — APPROVED** [review]

File: src/pages/Survey.tsx [hunk 2]
```diff
@@ -127,42 +130,62 @@ export const Survey = () => {
...
-      alignItems="center"
-      justifyContent="center"
-      flexDirection="column"
-    >
-      <Container mb={32} maxW="container.lg">
-        <Heading mb={6}>Professor Preferences Survey</Heading>
+    <>
+      {user != null && user.preferences.length > 0 ? (
+        <>
+          {" "}
```
Comment (Adamk8) [inline src/pages/Survey.tsx]: You can remove this {" "}, Sometimes if you have an extra space somewhere prettier puts this in, in this case I don't believe it changes anything

Comment (AndrewPolanyi) [reply src/pages/Survey.tsx]: good catch!

**Review by Adamk8 — APPROVED** [review]

**Pull request merged**